canonical/store-components
# Fix missing docs tab

## Done
Fixed the missing "Docs" tab

## QA
- Go to https://store-components-20.demos.haus
- Check that next to "Canvas", there is a "Docs" tab in the top left of the page
